ui/panels-plugin/src/plugins/trace-table/DataTable.tsx

```diff
@@ -184,7 +184,19 @@ function buildServiceStatsChips(
       sx={{ marginTop: '5px', marginRight: '5px' }}
```

Comment:
```suggestion
      sx={{ marginTop: '5px', marginRight: '5px', borderColor: serviceColorGenerator(serviceName) }}
```
Is it working btw? instead of adding `style`

Reply:
Yes, this works. If I'm adding it to `sx`, MUI creates a new CSS class for every color, which will impact performance: https://mui.com/system/getting-started/the-sx-prop/#dynamic-values

I noticed I missed this with the background color in `<Avatar>`, fixed now.

Reply:
OMG, didn't know about it 😮 We will need to do a pass on all component about it, specially about height/width property. I add this to my todo list 😄

Reply:
iirc I once had it on the `width` property on the Gantt chart with a resize observer hook, and when I slowly resized the browser window (with the dev tools inspector open) it got a bit laggy and I realized what's happening :sweat_smile: